source/_posts/2023-05-02-php-core-roundup-12.md

```diff
@@ -0,0 +1,265 @@
+- [Expansion of PHP Symbols?](https://externals.io/message/120094)
+- [Future stability of PHP?](https://externals.io/message/119834)
+- [Moving PHP internals to GitHub](https://externals.io/message/119955)
+- [Brainstorming idea: inline syntax for lexical (captured) variables](https://externals.io/message/119696)
+- [Possible RFC: `$_SERVER['REQUEST_TIME_FLOAT']` ](https://externals.io/message/119948)
+- [[RFC] PHP Technical Committee](https://externals.io/message/119829)
+- [Array spread append ](https://externals.io/message/119818)
+- [PHP Modules ](https://externals.io/message/119893)
+- [Property Hooks Discussion ](https://externals.io/message/119807)
+
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements. There are automated unit and integration tests for each of these changes, the PHP core developers review all pull requests.
+ 
+---
+
+### Full list of commits since [PHP Core Roundup #11](/blog/2023/04/01/php-core-roundup-11/)
+
+<details markdown="1">
+  <summary>Click here to expand</summary>
+
+ - Fix [GH-8841](https://github.com/php/php-src/issues/8841): php-cli core dump calling a badly formed function in [GH-10989](https://github.com/php/php-src/pull/10989) by Niels Dossche
+ - Fix [GH-10990](https://github.com/php/php-src/issues/10990): `mail()` throws `TypeError` after iterating over `$additional_headers` array by reference in [GH-10991](https://github.com/php/php-src/pull/10991) by Niels Dossche
+ - Fix [GH-10983](https://github.com/php/php-src/issues/10983): State-dependant segfault in `ReflectionObject::getProperties` in [GH-10984](https://github.com/php/php-src/pull/10984) by Niels Dossche
+ - Remove `XFAIL` from test cases for `mb_strcut` when used with JIS or ISO-2022-JP encoding in [c211e67b4e](https://github.com/php/php-src/commit/c211e67b4e) by Alex Dowad
+ - Add `ASAN XLEAK` support in [GH-10996](https://github.com/php/php-src/pull/10996) by Ilija Tovilo 💜
+ - Fix `add_function_array()` assertion when op2 contains op1 in [GH-10975](https://github.com/php/php-src/pull/10975) by Ilija Tovilo 💜
+ - Add `zend_alloc` `XLEAK` support in [GH-10999](https://github.com/php/php-src/pull/10999) by Ilija Tovilo 💜
+ - Tracing JIT: Fixed incorrect code generation fofr SEND-ing of result of ASSIGN to typed reference in [8a749c79d0](https://github.com/php/php-src/commit/8a749c79d0) by Dmitry Stogov
+ - Fix incorrect error handling in `dom_zvals_to_fragment()` in [GH-10981](https://github.com/php/php-src/pull/10981) by Niels Dossche
+ - Add forgotten upgrading note for the Readonly amendments RFC in [0a169cbd5b](https://github.com/php/php-src/commit/0a169cbd5b) by Máté Kocsis 💜
+ - Remove name field from the `zend_constant` struct ([#10954](https://bugs.php.net/bug.php?id=10954)) in [f42992f580](https://github.com/php/php-src/commit/f42992f580) by Máté Kocsis 💜
```

Comment:
It seems that some PR entries, such as this one, link to the bugs.php.net bugtracker (to an unrelated bug report) instead of the GitHub PR page.

Reply:
I force-pushed with all incorrect bugsnet links changed to GitHub PRs. Thank you @nielsdos.